openassessment/templates/openassessmentblock/edit/oa_edit_staff_assessment.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n %}
+{% spaceless %}
+<li class="openassessment_assessment_module_settings_editor" id="oa_staff_assessment_editor">
+    <div class="drag-handle action"></div>
+    <div class="openassessment_inclusion_wrapper">
+        <input type="checkbox" id="include_staff_assessment"
```

Comment:
this control is inaccessible. beyond scope of this PR.  new ticket here: https://openedx.atlassian.net/browse/ECOM-3389